cmd/juju/cloud/list.go

```diff
@@ -28,17 +28,27 @@ type listCloudsCommand struct {
 // listCloudsDoc is multi-line since we need to use ` to denote
 // commands for ease in markdown.
 var listCloudsDoc = "" +
-	"Provided information includes 'cloud' (as understood by Juju), cloud\n" +
-	"'type', and cloud 'regions'.\n" +
-	"The listing will consist of public clouds and any custom clouds made\n" +
-	"available through the `juju add-cloud` command. The former can be updated\n" +
-	"via the `juju update-clouds` command.\n" +
-	"By default, the tabular format is used.\n" + listCloudsDocExamples
+        "Output includes fundamental properties for each cloud known to the\n" +
+        "current Juju client. These are: name, number of regions, default region,\n" +
+	"type, and description.\n" +
+        "\nThe cloud name is what's used to refer to a cloud in any Juju command.\n" +
```

Comment:
Pretty sure that this sentence is not needed. It's obvious that cloud name is a cloud name.